README.md

```diff
@@ -7,4 +7,4 @@ This Prosemirror plugin adds the ability to add embed nodes to a document.
 This project can be tested locally using yarn: 
 
 `yarn` installs and sets up the project dependencies
-`yarn dev` builds the project locally, watches for file changes, and serves the project locally.
\ No newline at end of file
+`yarn start` builds the project locally, and watches for file changes. Open `dist/index.html` to view the test site.
```

Comment:
This is not currently working, there is nothing in place bundle the HTML.

Reply:
Are you certain? When I drag index.html into Chrome, things render as expected, and a bundle.js is generated in `/dist` – happy to pair, I may well be missing something.

(Aware this is not the usual way of serving these sites – until we have a need for a dev server, I thought just opening the file and refreshing the page when it changes would be enough)

Reply:
Oops, discussed out of band and I've left index.html out of dist. Thanks for spotting! Revised w/ a dev server in any case.